src/update_versions_to_latest_master.py

```diff
@@ -1,37 +1,18 @@
 #!/usr/bin/env python3
-from subprocess import run, PIPE
-
-
-def parse(line):
-    parts = line.split('\t')
-    return parts[1], parts[0]
-
-
-def get_version_for(repo_name, org="vimc"):
-    url = "git@github.com:{org}/{name}".format(org=org, name=repo_name)
-    text = run(["git", "ls-remote", url], check=True, stdout=PIPE, universal_newlines=True).stdout
-    pairs = list(parse(line) for line in text.split('\n') if line)
-    lookup = dict(pairs)
-    commit = lookup["refs/heads/master"]
-    return commit[:7]
+import os
+import pprint
+from os.path import join
+from subprocess import run
 
+import versions
 
 if __name__ == "__main__":
-    repo_map = [
-        ("api", "montagu-api"),
-        ("reporting_api", "montagu-reporting-api"),
-        ("db", "montagu-db"),
-        ("contrib_portal", "montagu-webapps"),
-        ("admin_portal", "montagu-webapps"),
-        ("report_portal", "montagu-webapps"),
-        ("cert_tool", "montagu-cert-tool"),
-        ("proxy", "montagu-proxy"),
-        ("orderly", "montagu-orderly")
-    ]
+    root = "submodules"
+    for submodule in os.listdir(root):
```

Comment:
Possibly simpler way of achieving this is to use:

```
git submodule update --recursive --remote
```

Reply:
Ah yes, that's much nicer.